dolphinscheduler-ui/src/views/projects/task/components/node/types.ts

```diff
@@ -91,7 +91,9 @@ interface ISwitchResult {
 }
 
 interface ISourceItem {
-  id: number
+  id?: number,
+  resourceName: string,
+  res?: string
```

Comment:
what's the meaning of `res`? Would it be better to use a full word？

Reply:
> what's the meaning of `res`? Would it be better to use a full word？

ya I agree that 'res' is not clear. It is kinda hard to change it because 'res' corresponds to 'res' field in the backend. For a temporal fix, I put a comment box there to illustrate what 'resourceName' and 'res' are.